### PR TITLE
Improve food registration.

### PR DESCRIPTION
**Pull request by RusTit**
Fix and improve #283 , #469 
Correct values
https://github.com/MatrexsVigil/harvestcraft/blob/b146e14fc53614c783e6c9d51302fcfbc99e5fa4/src/main/java/com/pam/harvestcraft/item/GeneralOreRegistry.java#L102
But in current branch "crop" string is prepend.

### CODE REVIEW

**Comment by WesTroy** [comment]
@RusTit - great work on everything lately, thanks!
Two little things:
- All your oreDict entries are appending "crop*" to the beginning (so, croplistAllfruit).
- Nutmeg is not a nut =) It's a spice from a tree.

**Comment by RusTit** [comment]
@WesTroy 
1. I know and I fix it with this request (look at file changes with this PR). "crop" naming is used as it correct Binnie naming, but Harvestcraft keys will be correct
2. This PR fix this too.

File: extratrees/src/main/java/binnie/extratrees/items/Food.java [hunk 1]
```diff
@@ -448,9 +453,14 @@ public void addOil(final int time, final int amount, final int mulch) {
 		RecipeManagers.squeezerManager.addRecipe(time, this.get(1), Fluids.SEED_OIL.getFluid(amount), Mods.Forestry.stack("mulch"), mulch);
 	}
 
-	protected Food registerOre(String string) {
-		OreDictionary.registerOre("crop" + string, this.get(1));
-		this.ores.add("crop" + string);
+	protected Food registerCrop(final String string) {
+	    final String name = "crop" + string;
+        this.ores.add(name);
```
Comment (mezz) [inline extratrees/src/main/java/binnie/extratrees/items/Food.java]: Please fix the weird mix of tabs and spaces in this method

**Review by mezz — CHANGES_REQUESTED** [review]

**Comment by RusTit** [comment]
I made an autoindend.
There's just really a mess of space and tabs.

**Comment by mezz** [comment]
Cool, thanks. You are already editing this file a lot so it's a good time to fix the mixed spaces.

**Review by mezz — APPROVED** [review]

**Pull request merged**